### DISCUSSION TITLE
Setting Dart version in ORT

### DISCUSSION THREAD

**Discussion by dodinh** [oss-review-toolkit/ort#9329] [Q&A] [answered]
Hi everyone,

Is it possible to set the Dart version used in ORT (especially in a Docker container)?

For **GradleInspector**, we can set the Java version using `javaVersion` or `JAVA_HOME`.
For **pub** I couldn't find anything related like `dartVersion`. I looked in the reference `config.yml` and searched in the repository.

**Comment by dodinh** [comment, marked as answer]
Nevermind, since I am working with 3rd party projects and have never done any Flutter development, I misunderstood the relation between Flutter and Dart.

Answer: use `flutterVersion` which automatically uses a compatible Dart version (see https://docs.flutter.dev/release/archive).

**Reply by sschuberth** [reply]
For reference, here's the full list of options currently supported by the Pub analyzer:

https://github.com/oss-review-toolkit/ort/blob/419b42b6dc21c1d1a41931aa24b338fe623d882f/plugins/package-managers/pub/src/main/kotlin/Pub.kt#L98-L102
